packages/ckeditor5-show-blocks/tests/manual/showblocks.html

```diff
@@ -0,0 +1,54 @@
+<div id="editor">
```

Comment:
If this manual test is to be of any use for devs and testers, I say we pump it up with a lot of rich content. 

For the editor:
* I'd go with GHS in the all-in configuration.
* I'd load lots of plugins/features, e.g. copy paste of https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html.

For the content:
* I'd start with https://ckeditor.com/docs/ckeditor5/latest/features/document-outline.html 
* I'd mix other guides' content into it to see how different widgets behave.
* I'd mix some arbitrary GHS content (from guides, manual tests) to see how structures such as `div > div > div` look like that would otherwise not be created by core editor features.

Don't spend 2h on this, it does not have to make sense as a document. All it needs to do is stress-test CSS we're gonna provide with the feature.